front/src/common/SpeedLimitByTagSelector/SpeedLimitByTagSelector.tsx

```diff
@@ -30,29 +30,23 @@ export default function SpeedLimitByTagSelector({
   );
 
   if (!speedLimitsTagsList.length) return null;
+
   return (
     <div className="osrd-config-item mb-2">
       <div
         className={cx('osrd-config-item-container', {
           'd-flex align-items-center gap-10': condensed,
         })}
       >
-        <img width="32px" src={icon} alt="speedometer" />
-        <span className="text-muted">{t('speedLimitByTag')}</span>
-        <SelectImprovedSNCF
-          sm
+        <Select
           disabled={disabled}
-          withSearch
-          dataTestId="speed-limit-by-tag-selector"
-          value={speedLimitByTag || t('noSpeedLimitByTag').toString()}
-          options={speedLimitsTagsList}
+          id="speed-limit-by-tag-selector"
+          value={speedLimitByTag || ''}
+          label={t('speedLimitByTagAbbrev')}
           onChange={(e) => {
-            if (e && e !== t('noSpeedLimitByTag')) {
-              dispatchUpdateSpeedLimitByTag(e);
-            } else {
-              dispatchUpdateSpeedLimitByTag(null);
-            }
+            dispatchUpdateSpeedLimitByTag(e || null);
```

Comment:
We no longer filter the `noSpeedLimitByTag` value, meaning when we select it, the store is updated with that and when adding/editing the train, we send this non consistent value instead of null. I'm afraid it might result to some weird things in the simulation. Maybe we should leave it like it was before ?